File: src/main/scala/intricateengineers/intricatemachinery/api/module/ModuleModel.java
package intricateengineers.intricatemachinery.api.module;

import intricateengineers.intricatemachinery.api.client.QuadHandler;

import net.minecraftforge.fml.relauncher.Side;
import net.minecraftforge.fml.relauncher.SideOnly;

public abstract class ModuleModel extends ModelBase {
    @SideOnly(Side.CLIENT)
    private QuadHandler quadHandler;

    @SideOnly(Side.CLIENT)
    public QuadHandler getQuadHandler(){
        if (this.quadHandler == null) {
            this.quadHandler = new QuadHandler(this);
        }    
        return this.quadHandler;
    }
}

File: src/main/scala/intricateengineers/intricatemachinery/api/model/ModuleModel.scala
package intricateengineers.intricatemachinery.api.model

import net.minecraftforge.fml.relauncher.SideOnly
import net.minecraftforge.fml.relauncher.Side
import intricateengineers.intricatemachinery.api.client.QuadHandler

abstract class ModuleModel extends ModelBase {

  @SideOnly(Side.CLIENT)
  lazy val quadHandler = new QuadHandler(this)

}
